lib/albacore/task_types/test_runner.rb

```diff
@@ -78,9 +85,9 @@ class Cmd
 
       # expects both parameters and executable to be relative to the
       # work_dir parameter
-      def initialize work_dir, executable, parameters, file, clr_command = true
+      def initialize work_dir, executable, parameters, files, clr_command = true
         @work_dir, @executable = work_dir, executable
-        @parameters = parameters.to_a.unshift(file)
+        @parameters = files.concat(parameters.to_a)
```

Comment:
Since this is a public API, could we just check if it's enumerable and then concat it; otherwise expect it to be a single file?